frontend/src/metabase-lib/expressions/helper-text-strings.unit.spec.ts

```diff
@@ -76,6 +75,10 @@ describe("getHelpText", () => {
 function setup(dbOpts?: Partial<Database>) {
   const database = createMockDatabase(dbOpts);
   const metadata = createMockMetadata({ databases: [database] });
+  const instance = metadata.database(database.id);
+  if (!instance) {
+    throw new TypeError();
+  }
```

Comment:
Is there any reason we're throwing a new error here? In another file, we also only cast the type to be not null. Should be consistent?

https://github.com/metabase/metabase/blob/d18295ff31c8173ed750b157bb8842cb950078b9/frontend/src/metabase-lib/expressions/__support__/shared.js#L295-L303